src/pods/canvas/canvas.pod.tsx

```diff
@@ -18,8 +18,15 @@ import {
 import { CanvasSvgComponent } from './canvas-svg.component';
 import { EditRelationPod } from '../edit-relation';
 import { mFlix } from './m-flix.mock.data';
+import useAutosave from '@/core/autosave/autosave.hook';
 
 export const CanvasPod: React.FC = () => {
+  const { stopAutosave } = useAutosave();
+
+  React.useEffect(() => {
+    return stopAutosave;
+  }, []);
```

Comment:
Not sure if we can even place this at app level

Reply:
Maybe we should expose a

`startAutosave` and `stopAutosave`

So it could be something like

```diff
React.useEffect(() => {
  startAutosave;

  return () => stopAutosave();
}, []);
```